src/components/Modal/Modal.css

```diff
@@ -1,28 +1,23 @@
 @import '../../styles/theme/index.css';
 
-:root {
-  --modal-border: solid 1px black;
-  --modal-container-background-color: rgba(0, 0, 0, 0.1);
-}
 
 .container {
-  display: none;
-  background-color: var(--modal-container-background-color);
-  display: flex;
-  justify-content: center;
-  align-items: center;
+  display: var(--modal-container-display, flex);
```

Comment:
you're moving from none to flex by default, is there a reason for that?

Reply:
hey @branquinhoaa can you please clarify this doubt?